src/util/metadata/side_metadata/global.rs

```diff
@@ -12,6 +12,18 @@ use std::fmt;
 use std::io::Result;
 use std::sync::atomic::{AtomicU8, Ordering};
 
+#[cfg(feature = "vo_bit")]
```

Comment:
It's just not right to put VO bit specific functions in `side_metadata::global`.  I suggest we make the `vo_bit` module public, and mark functions not intended to be public as `pub(crate)`.  It is actually part of the plan to make `vo_bit` public.  See: https://github.com/mmtk/mmtk-core/issues/804